### DISCUSSION TITLE
How to install the nomacs image viewer?

### DISCUSSION THREAD

**Discussion by idanej** [nomacs/nomacs#1028] [Q&A] [answered]
Hello, and excuse me for being a tech-illiterate github user. You can delete this thread later if I'm breaking any rules.

I've been a very pleased Windows 10 user of the nomacs image viewer for a few years. I have a new hard drive and have to reinstall a lot of software. I think I got nomacs from the website the last time I installed it, but it's no longer available. Can someone tell me how to install it in simple language? 

I do not think I had to install anything else to make it work the last time, so I'm confused by this bit (at https://github.com/nomacs/nomacs):

"We assume you have an IDE (i.e. Visual Studio), python, git, and [Qt](https://www.qt.io/download-open-source) installed."

**Comment by novomesk** [comment, marked as answer]
https://github.com/nomacs/nomacs/releases/download/3.17.2287/nomacs-setup-x64.msi

**Reply by idanej** [reply]
Thank you!
